src/main/java/org/threadly/litesockets/utils/SSLProcessor.java

```diff
@@ -116,17 +116,18 @@ private ByteBuffer getDecryptedByteBuffer() {
   }
 
   public MergedByteBuffers encrypt(final ByteBuffer buffer) {
+    return encrypt(new MergedByteBuffers(false, buffer));
+  }
+  
+  public MergedByteBuffers encrypt(final MergedByteBuffers lmbb) {
     final MergedByteBuffers mbb = new MergedByteBuffers(false);
     if(!startedHandshake.get()){
-      mbb.add(buffer);
+      mbb.add(lmbb);
```

Comment:
Could we not just return the passed in `lmbb` in this case?  If so we should move the mbb construction to after this check.